packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -186,11 +216,18 @@ export default function MongoNav({
     if (mode === Mode.Dev) {
       getDataFixtures().then(data => setData(data as DataInterface));
     } else {
-      getProductionData()
+      const body =
+        activeProjectId || activeOrgId
+          ? activeProjectId
+            ? { activeProjectId }
```

Comment:
You can send both values to the server - `{ activeProjectId, activeOrgId }`. The server will ignore orgId if projectId is given.

Reply:
👍